lib/matplotlib/figure.py

```diff
@@ -433,10 +432,10 @@ def show(self, warn=True):
             except NonGuiException:
                 pass
         if (backends._get_running_interactive_framework() != "headless"
-                and warn):
-            warnings.warn('Matplotlib is currently using %s, which is a '
-                          'non-GUI backend, so cannot show the figure.'
-                          % get_backend())
+                    and warn):
```

Comment:
Why the additional spaces? [PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation) example suggests:

~~~
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
~~~